source/module_hamilt_lcao/hamilt_lcaodft/FORCE_gamma.cpp

```diff
@@ -260,15 +260,15 @@ void Force_LCAO<double>::ftable(const bool isforce,
 
         if (PARAM.inp.deepks_out_unittest)
         {
-            LCAO_deepks_io::print_dm(dm_gamma[0], PARAM.globalv.nlocal, this->ParaV->nrow);
+            LCAO_deepks_io::print_dm(1, PARAM.globalv.nlocal, this->ParaV->nrow, dm_gamma); // 1 for gamma-only
 
             GlobalC::ld.check_projected_dm();
 
             GlobalC::ld.check_descriptor(ucell, PARAM.globalv.global_out_dir);
 
             GlobalC::ld.check_gedm();
 
-            GlobalC::ld.cal_e_delta_band(dm_gamma);
+            GlobalC::ld.cal_e_delta_band(dm_gamma,1);
```

Comment:
explain the meaning of '1'

Reply:
Here 1 is also nks in gamma only case, I will change it.